releaseNotes/MicrosoftExtensionsConfigurationAzureAppConfiguration.md

```diff
@@ -1,6 +1,20 @@
 ## Microsoft.Extensions.Configuration.AzureAppConfiguration
 ### [Package (NuGet)](https://www.nuget.org/packages/Microsoft.Extensions.Configuration.AzureAppConfiguration)
 
+### 6.0.0 - November 29, 2022
+* Fixed a bug where the configuration did not refresh when ChainedConfigurationProvider was used. [#168](https://github.com/Azure/AppConfiguration-DotnetProvider/issues/168)
+* Added the following new API for additional App Configuration geo-replication support. [#385](https://github.com/Azure/AppConfiguration-DotnetProvider/issues/385)
+
+   ```cs
+   public AzureAppConfigurationOptions Connect(IEnumerable<string> connectionStrings)
+   ```
+
+   The new API allows you to provide an ordered list of connection strings of your App Configuration store and its replicas.
+
+* Removed `IConfigurationRefresher.SetDirty` API. `IConfigurationRefresher.ProcessPushNotification` API should be used for push-model based configuration refresh. Refer to [this tutorial](https://docs.microsoft.com/en-us/azure/azure-app-configuration/enable-dynamic-configuration-dotnet-core-push-refresh?tabs=windowscommandprompt) for more details about the `ProcessPushNotification` API. [#357](https://github.com/Azure/AppConfiguration-DotnetProvider/issues/357)
+* Removed .NET 5 as a target framework. [#391](https://github.com/Azure/AppConfiguration-DotnetProvider/pull/391)
+* Removed support for Feature Management V2 schema introduced in Microsoft.FeatureManagement 3.0.0-preview library. [#315](https://github.com/Azure/AppConfiguration-DotnetProvider/issues/315)
```

Comment:
This never made it into any stable version release. We didn't mention the removal in the 5.2.0 release. We don't need to mention it here.

Reply:
I feel like we should've mentioned it in 5.2.0 release. We started tracking the release notes by differences between latest version and second latest version. This is the same pattern followed by SDK team ([release notes](https://github.com/Azure/azure-sdk-for-net/blob/Azure.Data.AppConfiguration_1.2.0/sdk/appconfiguration/Azure.Data.AppConfiguration/CHANGELOG.md)).

I think we digressed when we released 5.2.0 after 5.3.0-preview. But consider this - if we dont mention that FeatureMangement v2 schema was removed in 6.0.0, we would still need to mention it with the next 6.1.0-preview release, right?